packages/vkui/src/components/Search/Search.test.tsx

```diff
@@ -6,6 +6,7 @@ import { Search } from './Search';
 
 const getInput = () => screen.getByRole('searchbox');
 const getClearIcon = () => document.querySelector('.vkuiSearch__icon') as Element;
+const getFindButton = () => document.querySelector('.vkuiSearch__findButton') as Element;
```

Comment:
```suggestion
const getFindButton = () => document.querySelector<HTMLElement>(styles['.Search__findButton'])!;
```

в конце `!`, чтобы в `userEvent.click` TS не ругался 🌚